toolchains/lowrisc_toolchain_rv32imc/lowrisc_toolchain_gcc_rv32imc.bzl

```diff
@@ -0,0 +1,289 @@
+    for hdr_lib in ctx.attr.system_hdr_deps:
+        cc_ctx = hdr_lib[CcInfo].compilation_context
+        for inc in cc_ctx.system_includes.to_list():
+            command_line += ["-isystem", inc]
+    return command_line
+
+def _get_additional_system_include_paths(ctx):
+    include_paths = []
+    for hdr_lib in ctx.attr.system_hdr_deps:
+        cc_ctx = hdr_lib[CcInfo].compilation_context
+        for inc in cc_ctx.system_includes.to_list():
+            if inc not in ".":
+                include_paths.append(inc)
+    return include_paths
+
+def _lowrisc_toolchain_rv32imc_toolchain_config_info_impl(ctx):
+    tool_paths = [
+        tool_path(
+            name = "gcc",
+            path = "gcc_wrappers/{os}/gcc",
+        ),
+        tool_path(
+            name = "ld",
+            path = "gcc_wrappers/{os}/ld",
+        ),
+        tool_path(
+            name = "ar",
+            path = "gcc_wrappers/{os}/ar",
+        ),
+        tool_path(
+            name = "cpp",
+            path = "gcc_wrappers/{os}/cpp",
+        ),
+        tool_path(
+            name = "gcov",
+            path = "gcc_wrappers/{os}/gcov",
+        ),
+        tool_path(
+            name = "nm",
+            path = "gcc_wrappers/{os}/nm",
+        ),
+        tool_path(
+            name = "objcopy",
+            path = "gcc_wrappers/{os}/objcopy",
+        ),
+        tool_path(
+            name = "objdump",
+            path = "gcc_wrappers/{os}/objdump",
+        ),
+        tool_path(
+            name = "strip",
+            path = "gcc_wrappers/{os}/strip",
+        ),
+    ]
+    os = "nix"
+    postfix = ""
+    if ctx.host_configuration.host_path_separator == ";":
+        os = "windows"
+        postfix = ".bat"
+    tool_paths = [tool_path(name = t.name, path = t.path.format(os = os) + postfix) for t in tool_paths]
+
+    common_features = GetCommonFeatures(
+        compiler = "GCC",
+        architecture = ctx.attr.architecture,
+        float_abi = ctx.attr.float_abi,
+        endian = ctx.attr.endian,
+        fpu = ctx.attr.fpu,
+        include_paths = _get_additional_system_includes_command_line(ctx) +
+                        SYSTEM_INCLUDE_COMMAND_LINE +
+                        _get_injected_headers_command_line(ctx),
+        sysroot = None,
+    )
+    embedded_features = GetEmbeddedFeatures(
+        compiler = "GCC",
+        architecture = ctx.attr.architecture,
+        float_abi = ctx.attr.float_abi,
+        endian = ctx.attr.endian,
+        fpu = ctx.attr.fpu,
+    )
+    toolchain_config_info = cc_common.create_cc_toolchain_config_info(
+        ctx = ctx,
+        toolchain_identifier = ctx.attr.toolchain_identifier,
+        cxx_builtin_include_directories = SYSTEM_INCLUDE_PATHS,
+        host_system_name = "i686-unknown-linux-gnu",
+        target_system_name = "riscv32-unknown-elf",
+        target_cpu = ctx.attr.architecture,
+        target_libc = "unknown",
+        compiler = "riscv32-unknown-elf-gcc",
```

Comment:
Should this be `riscv32-unknown-elf-clang` ?

Reply:
This is a new rule to provide access to the old gcc configuration.

Reply:
Sorry, I left the comment on the wrong file.